src/components/Cart/CartFooter.tsx

```diff
@@ -0,0 +1,37 @@
+import CartPrice from './CartPrice';
+import styled from '@emotion/styled';
+import { CartPriceVariant } from './CartPrice';
+
+interface CartFooterProps {
+  price: number;
+  shippingFee: number;
+  totalPrice: number;
+}
+
+function CartFooter({ price, shippingFee, totalPrice }: CartFooterProps) {
+  const renderCartPrice = [
+    { title: '주문 금액', price, variant: 'default' as CartPriceVariant },
+    { title: '배송비', price: shippingFee, variant: 'shipping' as CartPriceVariant },
+    { title: '총 결제 금액', price: totalPrice, variant: 'total' as CartPriceVariant },
```

Comment:
```suggestion
    { title: '주문 금액', price, variant: 'default' as const },
    { title: '배송비', price: shippingFee, variant: 'shipping' as const },
    { title: '총 결제 금액', price: totalPrice, variant: 'total' as const },
```
타입 오류 때문이라면, 이렇게 표현할 수도 있답니다 ㅎㅎ

Reply:
반영했습니다ㅎㅎ

[refactor: as 타입 단언으로 타입 에러 해결 및 불필요 함수를 값으로 변경](https://github.com/woowacourse/react-shopping-cart/pull/405/commits/ab23fc05ac05933763e6edb02eec419c5ab3178a)